Dockerfile

```diff
@@ -127,6 +131,11 @@ RUN /opt/ort/bin/import_proxy_certs.sh && \
     mkdir -p $GOPATH/bin && \
     curl -ksS https://raw.githubusercontent.com/golang/dep/v$GO_DEP_VERSION/install.sh | sh && \
     curl -ksS https://raw.githubusercontent.com/commercialhaskell/stack/v$HASKELL_STACK_VERSION/etc/scripts/get-stack.sh | sh && \
+    # Install SDKs required for analysis.
+    curl -Os https://dl.google.com/android/repository/commandlinetools-linux-${ANDROID_SDK_VERSION}_latest.zip && \
+    unzip -q commandlinetools-linux-${ANDROID_SDK_VERSION}_latest.zip -d $ANDROID_HOME && \
+    rm commandlinetools-linux-${ANDROID_SDK_VERSION}_latest.zip && \
```

Comment:
Should we accept the licenses here, e.g: `yes | $ANDROID_HOME/tools/bin/sdkmanager --licenses`

Reply:
Is it needed? At least the Docker image builds fine without it.

Reply:
I believe it is needed for the issue mentioned in [here](https://stackoverflow.com/q/38096225/1127485).

Reply:
I'm aware of the basic issue, I just thought the Android Gradle plugin would auto-accept licenses when it downloads any missing SDK components. Also, our Android tests pass although we don't do the `--licenses` stuff in `.travis.yml`. But maybe our Android test projects are so minimalistic that they don't need any additional components to be installed. I can make that change to be on the safe side.

Reply:
It's fully up to you. I remembered myself spending some time on figuring this out, so I just wanted to share it, so that you can remember it in case you have an issue.

Reply:
I just checked the Travis build log, and the license prompt is [already being triggered](https://travis-ci.com/github/oss-review-toolkit/ort/builds/175042533#L462), which explains what the existing `yes |` is needed for. So, I think we're already good, or?

Reply:
I think so too.